pkg/datasource/handler.go

```diff
@@ -283,6 +286,14 @@ func (s *Service) exportFromVolume(parameters map[string]string) error {
 		return fmt.Errorf("can't get pod ip from environment variable")
 	}
 
+	storageIP := podIP
+	if ip, err := s.getLocalIPv4fromInterface(StorageNetworkInterface); err != nil {
+		s.log.WithError(err).Debugf("Failed to get IP from %v interface", StorageNetworkInterface)
```

Comment:
NIT: "..., fallback to use the default pod IP %v"

Reply:
Updated.